api/HTMLIFrameElement.json

```diff
@@ -142,66 +142,62 @@
           }
         }
       },
-      "allowFullScreen": {
+      "allowFullscreen": {
         "__compat": {
           "support": {
             "chrome": {
-              "prefix": "webkit",
-              "version_added": "17",
-              "notes": "Daily test builds only."
+              "version_added": "38"
             },
             "chrome_android": {
-              "prefix": "webkit",
-              "version_added": "18",
-              "notes": "Daily test builds only."
+              "version_added": "38"
             },
             "edge": {
-              "version_added": "≤18"
+              "version_added": "≤15"
```

Comment:
I couldn't go back further than Edge 15 on BrowserStack.

Reply:
> I couldn't go back further than Edge 15 on BrowserStack.

I also haven’t been able to find any way to get any closer than that, so I reckon that is close enough (and if/when somebody later able to get more-precise version info, the data can be further updated then)

Reply:
```diff
-             "version_added": "≤18"
+             "version_added": "≤15"
```
> I couldn't go back further than Edge 15 on BrowserStack.
>> I also haven’t been able to find any way to get any closer than that, so I reckon that is close enough (and if/when somebody later able to get more-precise version info, the data can be further updated then)

Actually, this change is causing the linter to fail, because `≤15` isn’t a valid/recognized version value for Edge. The closest allowed value for this case is actually `≤18`, so we need to just revert this part of the change.